index.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+const yargs = require('yargs');
+const loadConfig = require('./src/load-config');
+
+const { doPlan, doApply } = require('./src/execute');
+
+const params = yargs
+    .command('plan', 'Do plan to see if there are changes', {
+        config: {
+            alias: 'c',
+            description: 'config file',
+            type: 'string',
+        }
+    })
+    .command('apply', 'Apply changes', {
+        config: {
+            alias: 'c',
+            description: 'config file',
+            type: 'string',
+        }
+    })
+    .parse();
+
+const command = params._.pop() || '';
+if (!['plan', 'apply'].includes(command)) {
+    throw new Error(`Command ${command} is unrecognized`);
+}
+
+const config = loadConfig(params.config);
+
+if (command === 'plan') {
+    doPlan(config);
+}
+
+if (command === 'apply') {
+    doApply(config);
+}
```

Comment:
```suggestion
const config = loadConfig(params.config);
switch (command):
  case 'plan':
    doPlan(config);
    break;
  case 'apply': 
    doApply(config);
    break;
  default:
    throw new Error(`Command ${command} is unrecognized`);
}
```

sorry for not spotting earlier but this looks like a typical usecase for switch...case statement

Reply:
as discussed on vc, let's skip this idea